modules/running-insights-operator-gather-web-console.adoc

```diff
@@ -33,16 +33,20 @@ spec:
      state: Disabled
 ----
 +
---
-<1> Replace the `<your_data_gather>` with a unique name for your gather operation.
-<2> Enter individual gather operations to disable under the `gatherers` parameter. This example disables the `workloads` data gather operation and will run the remainder of the default operations. To run the complete list of default gather operations, leave the `spec` parameter empty. You can find the complete list of gather operations in the Insights Operator documentation.
---
-+
-. Click *Save*.
+<1> Under *metadata*, replace `<your_data_gather>` with a unique name for the gather operation.
+<2> Under *gatherers*, specify any individual gather operations that you intend to disable. In the example provided, `workloads` is the only data gather operation that is disabled and all of the other default operations are set to run.
+When the `spec` parameter is empty, all of the default gather operations run.
+
+[IMPORTANT]
+====
+Do not add a prefix of "periodic-gathering-" to the name of your gather operation because this string is reserved for other administrative operations and might impact the intended gather operation.
```

Comment:
```suggestion
Do not add a prefix of `periodic-gathering-` to the name of your gather operation because this string is reserved for other administrative operations and might impact the intended gather operation.
```

Reply:
The backticks would be needed here. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference